packages/account/src/Sections/Profile/PersonalDetails/personal-details-form.tsx

```diff
@@ -374,7 +374,7 @@ const PersonalDetailsForm = observer(() => {
                                             disabled={isFieldDisabled('phone')}
                                             data-testid='dt_phone'
                                         />
-                                        <VerifyButton />
+                                        <VerifyButton disable={dirty} />
```

Comment:
since its a boolean, can we rename to is_disabled?

Reply:
sure let me change it 😎